### PR TITLE
CatoNetworks - parse more fields

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**
https://github.com/SekoiaLab/integration/issues/1374

## Type of Change

- [ ] New intake format
- [X] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`poetry run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`poetry run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Extend the Cato SASE intake to parse additional application, API key, and traffic metadata from events.

New Features:
- Map Cato SASE application metadata, categories, Cato app name, request/response sizes, and source context into dedicated fields.
- Introduce parsing of API key audit events to capture the public API key name.

Enhancements:
- Improve firewall rule naming by falling back to the network rule name when the primary rule name is absent.
- Enrich events with destination domain and user agent information.

Tests:
- Add and update Cato SASE event samples, including new API key and antimalware fixtures, to cover the extended field mappings.

### CODE REVIEW

File: CatoNetworks/cato-sase/_meta/fields.yml [hunk 1]
```diff
@@ -13,11 +48,36 @@ cato.sase.event_type:
...
+  description: Name defined for the public API Key in the Cato Management Application
+  name: cato.sase.key_name
+  type: keyword
+
+cato.sase.request_size:
+  description: Request packet size in bytes
+  name: cato.sase.request_size
+  type: long
+
+cato.sase.response_size:
```
Comment (TOUFIKIzakarya) [inline CatoNetworks/cato-sase/_meta/fields.yml:61]: I think [http.response.bytes](https://www.elastic.co/docs/reference/ecs/ecs-http#field-http-response-bytes) is more suitable

File: CatoNetworks/cato-sase/_meta/fields.yml [hunk 2]
```diff
@@ -13,11 +48,36 @@ cato.sase.event_type:
   name: cato.sase.event_type
   type: keyword
 
+cato.sase.key_name:
+  description: Name defined for the public API Key in the Cato Management Application
+  name: cato.sase.key_name
+  type: keyword
+
+cato.sase.request_size:
```
Comment (TOUFIKIzakarya) [inline CatoNetworks/cato-sase/_meta/fields.yml:56]: I think [http.request.bytes](https://www.elastic.co/docs/reference/ecs/ecs-http#field-http-request-bytes) is more suitable.

**Review by TOUFIKIzakarya — COMMENTED** [review]
Thanks for the improvement.
Some small changes to do.

**Review by mchupeau-sk — APPROVED** [review]
LGFM
